app/helpers/application_helper.rb

```diff
@@ -0,0 +1,11 @@
+module ApplicationHelper
+    # Returns the full title on a per-page basis.
+    def full_title(page_title = '')
+    base_title = "Ruby on Rails Tutorial Sample App"
```

Comment:
indent
```suggestion
      base_title = "Ruby on Rails Tutorial Sample App"
```
tương tự các file khác nhé